data/Language/Lua.yaml

```diff
@@ -0,0 +1,31 @@
+identifier: Lua
+creation_date: 1993
+full_name: Lua
+wikidata: Q207316
+grammar_file: https://github.com/antlr/grammars-v4/blob/master/lua/Lua.g4
+extensions:
+  - lua
+datatypes:
+  - bool_lower_case
+keywords:
+  - and
+  - break
+  - do
+  - else
+  - elseif
+  - end
+  - false
+  - for
+  - function
+  - if
+  - in
+  - local
+  - nil
+  - not
+  - or
+  - repeat
+  - return
+  - then
+  - true
```

Comment:
Please remove this as well. And a small suggestion to remember. Whenever you add values like `true` or `false` or `on` or `off` , please make sure that you embed them in `' '` , This is because `yaml` considers these values as `0` or `1` . Like for eg: `true`, `false`